### DISCUSSION TITLE
Feature Request: Add count for selected items

### DISCUSSION THREAD

**Discussion by sergeifilippov** [craftcms/cms#8557] [Ideas]
It would be **VERY** useful to add a count of how many items you have selected in the Assets sections. Something like this:

<img width="895" alt="assets_-_richmond_mall" src="https://user-images.githubusercontent.com/123676/41289162-ef070c08-6e9c-11e8-86e3-2d4355e145cf.png">

**Comment by brandonkelly** [comment]
We should look into this alongside #2516.
